actions/plugins/frontend/action.yml

```diff
@@ -8,9 +8,17 @@ runs:
       shell: bash
       run: ${{ github.action_path }}/pm.sh install
 
+    - name: Lint
+      shell: bash
+      run: ${{ github.action_path }}/pm.sh lint
+
+    - name: Typecheck
+      shell: bash
+      run: ${{ github.action_path }}/pm.sh typecheck
```

Comment:
Those two were not present in Drone. Is it safe to always attempt to run them? Will/should they always be defined in `package.json`, or do we have to add a check to ensure that the `lint` and `typecheck` are defined in `package.json` ?

They are scaffolded by create-plugin: https://github.com/grafana/plugin-tools/blob/66052204455d8d139ee005d8586e447ed61a213e/packages/create-plugin/templates/common/_package.json#L4-L17

I want to double-check because this is technically a breaking change from Drone

Reply:
@jackw ptal :slightly_smiling_face: